### ISSUE TITLE
half scanned area

### ISSUE THREAD

**Issue by puredota** [RocketMap/RocketMap#1549]
![scanned](https://cloud.githubusercontent.com/assets/20898453/20333776/899c9504-ab7a-11e6-9d02-a4079e567f68.jpg)

just updated
now the scanned area is not circle by circle, it become half by half, 
0 pokemon, 0 gym, 0 stop detected
What is wrong?

**Comment by BradAF** [comment]
Same issue -- cloned from git repository, copied in old config and verified accounts do not have captcha waiting by logging into phone. Everything successfully logs in, no errors shown but get 0 pokemon, 0 gym and 0 stop detected.

**Comment by he3amo** [comment]
@puredota [I wrote about it](https://github.com/PokemonGoMap/PokemonGo-Map/issues/1542). Try one acc with step 3 just for test.

**Comment by divnotded** [comment]
The half by half is correct as the workers never jump from the outer ring back to the inner ring. Implemented months ago. https://github.com/PokemonGoMap/PokemonGo-Map/pull/1010. The 0-0-0 is not due to path of workers.

**Comment by sebastienvercammen** [comment]
Closing this issue because it's very old and no longer relevant to the current codebase. If the issue still persists for you today, comment on this issue to reopen it with new details (logs, screenshots).
